packages/lab/src/components/Flow/DroppableFlow.tsx

```diff
@@ -224,10 +225,14 @@ export const HvDroppableFlow = ({
       const nds = applyNodeChanges(changes, nodes);
       setNodes(nds);
 
-      handleFlowChange(nds, edges);
-      onNodesChangeProp?.(changes);
+      if (!initialRender.current) {
```

Comment:
This way, if we have an empty canvas and add one node to it, the `onFlowChange` won't be triggered. I believe setting the `ref` when the component's mount as before would solve this.

Reply:
Yeah, I'll keep the `ref` but revert back to the original solution. The `useIsMounted` hook doesn't work in what we're trying to achieve here.